woocommerce/woocommerce-android
# Merge IAP feature to the trunk

### Description


### Testing instructions

This is a merge of all IAP-related code to the trunk. The code was reviewed with different levels of attention so feel free to make any proposal

Also, if something not clear, please left your comments - this will help me to write a better documentation

The main goal here is to make sure that we won't let Google Play know that we have IAP in the app, so please think of anything that might be missing regarding this

### Images/gif

To run showcase activity set the Launch Option as follows:
<img width="721" alt="image" src="https://user-images.githubusercontent.com/4923871/201850705-e5b3958b-bd7c-450c-a5cd-dbd9b93381e5.png">
`com.woocommerce.android.iapshowcase.IAPShowcaseActivity` and use `vanillaDebug` build type

More info on how to test IAP:   PdfdoF-1Ej-p2


https://user-images.githubusercontent.com/4923871/201852268-faffb472-d787-464c-a2b6-e0498bb61d01.mp4
